src/test/java/bio/terra/service/auth/iam/IamServiceTest.java

```diff
@@ -257,4 +257,41 @@ void testGetGroup() throws InterruptedException {
     when(iamProvider.getGroup(accessToken, groupName)).thenReturn(groupEmail);
     assertEquals(groupEmail, iamService.getGroup(groupName));
   }
+
+  @Test
+  void testGetGroupPolicyEmails() throws InterruptedException {
+    String groupName = "groupName";
+    String policyName = IamRole.MEMBER.toString();
+    String accessToken = "accessToken";
+    when(googleCredentialsService.getApplicationDefaultAccessToken(any())).thenReturn(accessToken);
+    when(iamProvider.getGroupPolicyEmails(accessToken, groupName, policyName))
+        .thenReturn(new ArrayList<>());
+    assertEquals(iamService.getGroupPolicyEmails(groupName, policyName), new ArrayList<>());
+  }
+
+  @Test
+  void testAddEmailToGroup() throws InterruptedException {
+    String groupName = "groupName";
+    String policyName = IamRole.MEMBER.toString();
+    String email = "user@gmail.com";
+    String accessToken = "accessToken";
+    when(googleCredentialsService.getApplicationDefaultAccessToken(any())).thenReturn(accessToken);
+    when(iamProvider.addGroupPolicyEmail(accessToken, groupName, policyName, email))
+        .thenReturn(new ArrayList<>(List.of("user@gmail.com")));
```

Comment:
I think all the `new ArrayList<>()` calls here can be removed or replaced with `List.of()`. In general we prefer using `List.of()` as it's easier to read and write, and is also slightly more memory efficient. One reason to not use `List.of()` is that the list it returns is read-only, so it can't be used in all situations.